### ISSUE TITLE
show/visualize full texts in Argumentview

### ISSUE THREAD

**Issue by justelex** [Qwlouse/Findeco#339]
In Argumentview the text is cutted of after a count of chars. We should:
-Indicate that this is a cutted off text
-Add an arrow or Something to get to the Full text (in Addiditon to Title klick)

**Comment by pinae** [comment]
You can always navigate to the argument and show the full text there. The easiest way to go there is by clicking the Heading which is a link.

**Comment by justelex** [comment]
Yes shure, but it is not really intuitive and we should change this behaviour
